lib/travis/build/script/julia.rb

```diff
@@ -67,7 +67,7 @@ def setup
             when 'windows'
               sh.cmd %Q{curl -A "$CURL_USER_AGENT" -sSf -L --retry 7 -o julia-installer.exe '#{julia_url}'}
               sh.cmd 'chmod +x julia-installer.exe'
-              if config[:julia] == 'nightly' || Gem::Version.new(config[:julia]) >= Gem::Version.new('1.4')
+              if config[:julia] =~ /^(?:nightly|1)$/ || Gem::Version.new(config[:julia]) >= Gem::Version.new('1.4')
```

Comment:
```suggestion
              if config[:julia] =~ /^(?:nightly|\d\+)$/ || Gem::Version.new(config[:julia]) >= Gem::Version.new('1.4')
```
Future-proofing for versions after 1 :)

Reply:
Perhaps, when Julia 2.0 is released, more drastic fixes will be required. :smile:
I think it is readable to explicitly specify "1" since "2" is invalid for now.

Reply:
FWIW this is a serious suggestion; I'd rather not have something that we know will eventually be wrong, even if that eventuality is not soon, especially since the fix is just a few extra characters.

Reply:
:confused:
```ruby
irb(main):001:0> Gem::Version.new("2") >= Gem::Version.new("1.4")
=> true
```

Reply:
Yeah, that's the behavior we want: we want to use the new installer setup for anything 1.4 or later.

Reply:
Right, so numerals `2` and up will be taken care of by `Gem::Version`. A bare `1` is a little unsightly, but having it explicit seems OK to me.

Reply:
Yes, I mean:
```ruby
irb(main):001:0> config = {:julia => '2'}
=> {:julia=>"2"}
irb(main):002:0> config[:julia] =~ /^(?:nightly|1)$/ || Gem::Version.new(config[:julia]) >= Gem::Version.new('1.4')
=> true
```

The "1" is a special case like "nightly", and other `/\d+/` are not.